lib/presentation/widget/shared_space/file_surfing/workgroup_nodes_surfing_viewmodel.dart

```diff
@@ -496,4 +508,66 @@ class WorkGroupNodesSurfingViewModel extends BaseViewModel {
         .actionText(AppLocalizations.of(context).cancel)
         .build());
   }
+
+  void openPopupMenuSorter(BuildContext context, List<Widget> actionTiles) {
+    ContextMenuBuilder(context)
+        .addHeader(SimpleBottomSheetHeaderBuilder(Key('order_by_menu_header'))
+        .addLabel(AppLocalizations.of(context).order_by)
+        .build())
+        .addTiles(actionTiles)
+        .build();
```

Comment:
Are you sure that you need 2 `build()` ?

Reply:
Cool, it fixed, it